### ISSUE TITLE
Directories <–> Breadcrumbs

### ISSUE THREAD

**Issue by RaoOfPhysics** [cernopendata/opendata.cern.ch#1131]
Replace 404 pages under directories with appropriately listed index pages pointing to the next level in the navigation.

So, http://opendata.cern.ch/visualise/ should list all relevant pages such as "events" and "histograms", while http://opendata.cern.ch/visualise/events/ (e.g.) should point to the appropriate event-visualisation tools where available.

**Comment by suenjedt** [comment]
@daslerr @sefeg @hjhsalo to be checked in the new Open Data on Invenio 3

**Comment by sefeg** [comment]
We can of course - for some specific URL's (/directories) - define such pages. I am wondering however, how relevant that is...how many people are trying to access a page based on an non-existing URL that they must have "guessed"? Maybe @tiborsimko could shed some light based on statistics?

**Comment by suenjedt** [comment]
@sefeg let's ask Stella for advice here (standards) before we start too much research.

**Comment by RaoOfPhysics** [comment]
Just FYI: we re-discussed this at yesterday's meeting, and there seemed to be support for the idea.

**Comment by sefeg** [comment]
Closing, as those pages exist by now. See for example, http://opendata-dev.web.cern.ch/software or http://opendata-dev.web.cern.ch/news - they jump right to the corresponding facet entries.

**Comment by RaoOfPhysics** [comment]
Reopening because this still needs addressing on some pages and needs reworking of the breadcrumbs text.

- The [`/visualise` page](http://opendata-dev.web.cern.ch/visualise) should show options for "Events" and "Histograms". Right now it shows nothing. <img width="1072" alt="firefox - - screen shot 28 nov 2017 at 13 34 36" src="https://user-images.githubusercontent.com/7623019/33320113-6b8f1104-d441-11e7-998f-4b5a7c22bbf7.png">
    - It should also have breadcrumbs that say: "Home / Visualise".
- The [`/visualise/events` page](http://opendata-dev.web.cern.ch/visualise/events) shows CMS and OPERA events, but the breadcrumbs need to be changed to "Home / Visualise / Events". <img width="1072" alt="firefox - - screen shot 28 nov 2017 at 13 34 23" src="https://user-images.githubusercontent.com/7623019/33320166-a608b40c-d441-11e7-9b67-6fc53071d3a9.png">
- The [`/visualise/events/cms` page](http://opendata-dev.web.cern.ch/visualise/events/cms) should have breadcrumbs that say "Home / Visualise / Events / CMS". <img width="1072" alt="firefox - - screen shot 28 nov 2017 at 13 34 12" src="https://user-images.githubusercontent.com/7623019/33320230-d014ff6c-d441-11e7-9b4f-63057cb69235.png">
- The same needs to be done for "Histograms" e.g., and any other URLs with sub-directories.
    - URL directories pointing to something meaningful is also useful for things like:
        - http://opendata-dev.web.cern.ch/articles/ (currently dead: can this point to search for articles?)

---

### Other notes

- See #1920: If it is approved, replace all "events" above with "collisions".
- Trailing slashes in URLs should be respected. [`/visualise/events/`](http://opendata-dev.web.cern.ch/visualise/events/) shows nothing while [`/visualise/events`](http://opendata-dev.web.cern.ch/visualise/events) does.

**Comment by ioannistsanaktsidis** [comment]
Closing as discussed that we are going to completely remove breadcrumbs. @annatrz  is taking care now.

**Comment by RaoOfPhysics** [comment]
Sorry, but I'm not sure removing breadcrumbs is the best idea here. There's clearly a hierarchy of information (and, indeed, of pages), e.g.: Visualise &rarr; Events &rarr; CMS.

**Comment by RaoOfPhysics** [comment]
@daslerr, @sefeg: Would you have some time to discuss the UI/UX aspects of this sometime tomorrow? Thanks!

**Comment by tiborsimko** [comment]
There were basically two options, either to complete the breadcrumbs, or to remove them. We have briefly discussed pros/cons, and since only a handful of pages use hierarchy, it seemed OK to remove them. But I fully understand that if the thumbnails are gone, the navigation will be harder in today's search-oriented page if people will happen to land on some sub-page... A related observation made by @katilp is whether people will find the event display page easily from some sub-page, now that the "Explore" is gone from the header menu. There are many options how to address these findings, e.g. add "Explore" for sub-pages but not on the home page since it would play double role, etc. There are several pros/cons with each choice, perhaps we can rethink the whole navigation experience from various angles with @sefeg and @daslerr now that we have new final prototype easily visible and testable?

**Comment by sefeg** [comment]
I agree that we can always revisit the discussion. As any discussion / implementation / testing requires considerable time, I think it is best if we consider this after the release. I think that for the moment we should stick with what has been decided. Possible enhancements can always be done afterwards. We could possibly meet next week @RaoOfPhysics ? (@daslerr @tiborsimko )

**Comment by RaoOfPhysics** [comment]
@sefeg: OK for next week.
